ksp_plugin/vessel.cpp

```diff
@@ -577,11 +577,11 @@ void Vessel::StartPrognosticatorIfNeeded() {
 }
 
 Status Vessel::RepeatedlyFlowPrognostication() {
-  for (;;) {
+  for (auto wakeup_time = std::chrono::steady_clock::now();;
```

Comment:
That initialization isn’t doing anything anymore (except deducing the type, but that’s a slightly confusing use of `now`).
`std::chrono::steady_clock::time_point wakeup_time;;`.

Reply:
Done.